python/cucim/src/cucim/core/operations/spatial/tests/test_random_flip.py

```diff
@@ -0,0 +1,55 @@
+import os
+import cupy
+import numpy as np
+from PIL import Image
+import skimage
+import cucim.core.operations.spatial as spt
+
+def get_input_arr():
+    img = skimage.data.astronaut()
+    arr = np.asarray(img)
+    arr = np.transpose(arr)
+    return arr
+
+def get_flipped_data():
+    img1 = Image.open(os.path.join(os.path.abspath(os.path.dirname(__file__)), "flipped.png"))
+    arr_o = np.asarray(img1)
+    arr_o = np.transpose(arr_o)
+    return arr_o
+
+def test_rand_flip_numpy_input():
+
+    arr = get_input_arr()
+    flip_arr = get_flipped_data()
+    output = spt.rand_image_flip(arr,prob=1.0,spatial_axis=(1,2))
+
+    assert np.allclose(output,flip_arr)
+
+def test_rand_flip_zero_prob():
+
```

Comment:
nit: blank line

Reply:
corrected and all test functions checked.